comp/metadata/inventoryagent/README.md

```diff
@@ -62,6 +62,9 @@ The payload is a JSON dict with the following fields
     `network_config.enabled` config option in `system-probe.yaml`).
   - `feature_oom_kill_enabled` - **bool**: True if the OOM Kill check is enabled for System Probe (see: `system_probe_config.enable_oom_kill` config option in `system-probe.yaml`).
   - `feature_tcp_queue_length_enabled` - **bool**: True if TCP Queue Length check is enabled in System Probe (see: `system_probe_config.enable_tcp_queue_length` config option in `system-probe.yaml`).
+  - `python` - **object**:
+    - `python_version` - **string**: The version of the embedded python.
```

Comment:
Do we need this ? Since we're shipping python, it's version will always be the same for a version of the Agent.

Reply:
This is not a hard requirement as we could guess it, but it's way easier to send it alongside the packages, so we have a reliable source of what's currently running within the agent